evm/src/cpu/columns.rs

```diff
@@ -139,6 +142,9 @@ pub struct CpuColumnsView<T> {
     /// If CPU cycle: the opcode, broken up into bits in **big-endian** order.
     pub opcode_bits: [T; 8],
 
+    /// If CPU cycle: The program counter for the next instruction.
+    pub next_program_counter: T,
```

Comment:
Is this needed? I would think we could have constraints interact directly with `nv.program_counter`, for logic like jumps etc.

(I see how that could be problematic if the next row is not a CPU cycle, but I think once we finish bootloading and reach CPU cycle rows, all subsequent rows would be CPU cycle rows?)

Reply:
I had the same thought. I decided to leave this column in here so this PR can show what the constraints end up being; I plan to remove it in a following PR